server/command/command.go

```diff
@@ -1062,6 +1068,39 @@ func (r *Runner) actionTest(args []string) {
 	}
 }
 
+func (r *Runner) actionTestGeneratePlaybooks(params []string) {
+	if len(params) < 1 {
+		r.postCommandResponse("The command expects one parameter: <numPlaybooks>")
+		return
+	}
+
+	numPlaybooks, err := strconv.Atoi(params[0])
+	if err != nil {
+		r.postCommandResponse("Error parsing the first argument. Must be a number.")
+		return
+	}
+
+	if numPlaybooks > 5 {
+		r.postCommandResponse("Cant generate playbooks since the number of playbook generated must be at most five")
+		return
+	}
+
+	playbookIds := make([]string, 0, numPlaybooks)
+	for i := 0; i < numPlaybooks; i++ {
+		dummyPlaybook := dummyListPlaybooks[i]
+		dummyPlaybook.TeamID = r.args.TeamId
+		newPlaybookID, err := r.playbookService.Create(dummyPlaybook, r.args.UserId)
+		if err != nil {
+			r.warnUserAndLogErrorf("There was an error while creating playbook. Err: " + err.Error())
```

Comment:
You can use a formatted string here, as follows:
```suggestion
			r.warnUserAndLogErrorf("unable to create playbook: %v",  err)
```